datanode/sqlsubscribers/position.go

```diff
@@ -112,11 +119,29 @@ func (p *Position) Push(ctx context.Context, evt events.Event) error {
 		return p.handleSettleMarket(ctx, event)
 	case tradeEvent:
 		return p.handleTradeEvent(ctx, event)
+	case ordersClosed:
+		return p.handleOrdersClosedEvent(ctx, event)
 	default:
 		return errors.Errorf("unknown event type %s", evt.Type().String())
 	}
 }
 
+func (p *Position) handleOrdersClosedEvent(ctx context.Context, event ordersClosed) error {
+	p.mutex.Lock()
+	defer p.mutex.Unlock()
+	// @TODO implement this
+	positions, err := p.store.GetByMarketAndParties(ctx, event.MarketID, event.Parties())
+	if err != nil {
+		return errors.Wrap(err, "error getting positions")
+	}
+	for _, pos := range positions {
+		pos.UpdateOrdersClosed()
+		// an update of an existing position here can't really fail
+		_ = p.updatePosition(ctx, pos)
```

Comment:
I get what you're saying but still be better to handle the error and return though, may that a future change results in error? method already returns error

Reply:
Method signature returns an error, but IIRC, it's just adding the entity to a batch, which actually doesn't return an error. I can add a check and return (or ignore?) the error. We should update as many positions as possible, but when that fails, I'm not sure whether we should continue trying